proto/v1/database_service.proto

```diff
@@ -62,6 +62,13 @@ service DatabaseService {
     option (google.api.http) = {get: "/v1/{name=instances/*/databases/*/schema}"};
   }
 
+  rpc DiffDatabaseSchema(GetDatabaseSchemaDiffRequest) returns (DatabaseSchemaDiff) {
```

Comment:
"DatabaseSchemaDiffResponse" to be consistent with other response message names.